theano/tensor/nnet/opt.py

```diff
@@ -245,6 +252,8 @@ def local_conv2d_gradweight_cpu(node):
         res = res[:, :, ::-1, ::-1]
 
     res = theano.tensor.patternbroadcast(res, node.outputs[0].broadcastable)
+
+    copy_stack_trace(node.outputs[0], res)
```

Comment:
Again, this one is fine, but I would also add the trace to the `res` defined on line 249.